pages/common/awk.md

```diff
@@ -29,8 +29,8 @@
 
 - Print all values starting from the third column:
 
-`awk '{ s = ""; for (i=3; i <= NF; i++) s = s $i " "; print s }'`
+`awk '{for (i=3; i <= NF; i++) printf $i""FS; print""}' {{filename}}`
 
 - Print different values based on conditions:
 
-`awk '{if ($1 == "foo") print "Exact match foo"; else if ($1 ~ "bar") print "Partial match bar"; else print "Baz"}'`
+`awk '{if ($1 == "foo") print "Exact match foo"; else if ($1 ~ "bar") print "Partial match bar"; else print "Baz"}' {{filename}}`
```

Comment:
Perhaps this will read better?
```suggestion
`awk '{if ($1 == "foo") print "Exact match foo"; else if ($1 ~ "bar") print "Partial match bar"; else print "Baz"}' {{path/to/filename}}`
```

Reply:
Wouldn't both of your suggestions make this example even longer?

Reply:
i kept the naming consistent with the rest of the `awk.md` file thus using `{{filename}}`. If you suggest to change it to path to file, then maybe the entire file should be changed?

Reply:
@zdroid: Yeah, but then it would be more consistent with other pages.

@vivekkrish: Ah, I didn't notice that. Perhaps it might be worth leaving it as-is in this case for now since the examples are quite long.